src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -610,6 +610,11 @@ private float BoundScrollPosition(float targetPosition)
             }
             else
             {
+                if (Math.Equals(maxScrollDistance, 0.0f))
```

Comment:
I think we should calculate maxScrollDistance when size of mScrollingChild is set or changed.

How about listening Relayout event of mScrollingChild and set it in callback?

Reply:
Yes, I think it's a good idea!
Could you make the patch?
I abandon this patch when you make the patch.

Reply:
God it. I'll request review after making my patch :)

Reply:
https://github.com/Samsung/TizenFX/pull/1665 << please review my patch